### ISSUE TITLE
Creating Classrooms - Subject, Description, and School ignored

### ISSUE THREAD

**Issue by shaunanoordin** [zooniverse/edu-api-front-end#111]
## Bug Needs Fixing
Issue: when creating a new Classroom (in I2A), any data specified in the 'Subject', 'School', or 'Description' fields are ignored - the Classroom is _only_ created with the 'Name' field intact.

### Analysis
Not sure why this is happening or when it started. I can see that the payload data for both Create New Classroom and Edit Existing Classroom actions _look OK._

```
//Create New Classroom
POST URL:https://education-api-staging.zooniverse.org/teachers/classrooms/
Payload: {"data":{"attributes":{"name":"Example 7 Dec","subject":"Test 1","school":"Test 2","description":"Test 3"},"relationships":{"program":{"data":{"id":"1","type":"programs"}}}}}

//Edit Existing Classroom
PUT https://education-api-staging.zooniverse.org/teachers/classrooms/38
Payload: {"data":{"attributes":{"name":"Example 7 Dec","subject":"Test A","school":"Test B","description":"Test C"}}}
```

### Status
Needs investigation and solution. Medium-priority. 🤷‍♂️

**Comment by srallen** [comment]
I haven't replicated this. Is it still happening?

**Comment by srallen** [comment]
Just noting that the bug is with the optional field values being set in the database. The POST payload on the client side looks correct and we get back a successful response, but the classroom does not have the subject and other fields saved.

**Comment by srallen** [comment]
This is fixed by zooniverse/education-api#84
